tests/test_rank.py

```diff
@@ -160,3 +160,105 @@ def test_order_label_issues_using_self_confidence_ranking():
     score_idx = sorted(list(zip(scores, indices)), key=lambda y: y[0])  # sort indices by score
     label_issues_indices2 = [z[1] for z in score_idx]
     assert all(label_issues_indices == label_issues_indices2)
+
+
+def test_order_label_issues_using_confidence_weighted_entropy_ranking():
+    label_issues_indices = rank.order_label_issues(
+        label_issues_mask=data["label_errors_mask"],
+        labels=data["labels"],
+        pred_probs=data["pred_probs"],
+        rank_by="confidence_weighted_entropy",
+    )
+    scores = rank.get_confidence_weighted_entropy_for_each_label(data["labels"], data["pred_probs"])
+    indices = np.arange(len(scores))[data["label_errors_mask"]]
+    scores = scores[data["label_errors_mask"]]
+    score_idx = sorted(list(zip(scores, indices)), key=lambda y: y[0])  # sort indices by score
+    label_issues_indices2 = [z[1] for z in score_idx]
+    assert all(label_issues_indices == label_issues_indices2)
+
+
+def test_order_label_issues_using_scoring_func_ranking():
+
+    # test all scoring methods with the scoring function
+    scoring_methods = ["self_confidence", "normalized_margin", "confidence_weighted_entropy"]
+
+    for method in scoring_methods:
+        label_issues_indices = rank.order_label_issues(
+            label_issues_mask=data["label_errors_mask"],
+            labels=data["labels"],
+            pred_probs=data["pred_probs"],
+            rank_by=method,
+        )
+        scores = rank.score_label_quality(data["labels"], data["pred_probs"], method=method)
+        indices = np.arange(len(scores))[data["label_errors_mask"]]
+        scores = scores[data["label_errors_mask"]]
+        score_idx = sorted(list(zip(scores, indices)), key=lambda y: y[0])  # sort indices by score
+        label_issues_indices2 = [z[1] for z in score_idx]
+        assert all(label_issues_indices == label_issues_indices2)
```

Comment:
when this fails, would be helpful to print which `scoring_method` it failed on.

Reply:
The lack of equivalency is the failure in this case. Any difference is failure for this test. Its unspecified an exactness for what the indices should be, only their equivalency is expected.

Reply:
The way to do this @JohnsonKuan  is not use a for-loop. This should be rewritten using @parameterize. See https://github.com/cleanlab/cleanlab/blob/master/tests/test_filter_count.py#L332 for an example.

Reply:
@cgnorthcutt Nice! That is a convenient decorator. Thanks for sharing. I rewrote the tests to use @parameterize

@jwmueller Agree - added messages to asserts so tests will print the scoring method if assertions fails

315e956b18f489a90b90a2ca62016b34176997fc
b56d2d8d969a1e8e8f91fd29eca706b2932edeb9